include/lsst/jointcal/Tripletlist.h

```diff
@@ -38,16 +38,20 @@ typedef Eigen::Triplet<double> Trip;
 // it would be wise to implement it differently if talking to cholmod
 class TripletList : public std::vector<Trip> {
 public:
+    using Index = Eigen::Index;
```

Comment:
Why use this here and not elsewhere (and/or the `vector<Index>` typedef)?

Reply:
What about this one? I'm not sure it gains us anything in this file (and I don't think it should be used more broadly than this file), and the `vector<Index>` type takes care of the most complicated use.